plugins/woocommerce-admin/client/core-profiler/actions/tracks.tsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { getSetting } from '@woocommerce/settings';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import {
+	CoreProfilerStateMachineContext,
+	UserProfileEvent,
+	BusinessInfoEvent,
+} from '..';
+import { POSSIBLY_DEFAULT_STORE_NAMES } from '../pages/BusinessInfo';
+
+const recordTracksStepViewed = (
+	_context: unknown,
```

Comment:
Thanks for addressing that, @rjchow! Should we also change the types here? Or do we have a purpose for using `unknown` since we don't use it?

Reply:
I think lint complains about implicit any if you don't give a type 🥲

Reply:
Oh, I was under the impression that `_context` was meant for `CoreProfilerStateMachineContext` at least on the other functions in here, is that true? 🤔

Reply:
I think maybe there's a bit of confusion here - there are two lint rules in place I suppose, 
1. unused variable - so adding _ infront denotes it as a known unused variable, and lint doesn't complain about that
2. typescript implicity any - if there's no type added then it's an implicit any, and lint complains. 

It's ok to type it as `CoreProfilerStateMachineContext` too, but since we don't use it, it seems unnecessary to add a type annotation here. 

I do need to take some time when it's available, to figure out exactly how to type these Assign Actions, they use a bunch of generics and I haven't quite figured out what to put in them.

Reply:
Ok, gotcha! Thanks for the explanation 👍